Update docs to mention adding target to paths.

drewverlee opened the pull request:
## Purpose
To document the requirements on Triangulum application users imposed by an [earlier change. 
](https://github.com/sig-gis/triangulum/commit/03053626b4d15ec1363f64d39257e41e4e54463c)

Before that change, it was already a requirement that the app.js be built to /target/public/cljs/app.js, the change was to include target on the classpath so the app.js could be included in the jar.


## Related Issues
none

## Submission Checklist
- [ ] Commits include the JIRA issue and the `#review` hashtag (e.g. `TRI-### #review <comment>`)
- [x] Code passes linter rules (`clj-kondo --lint src`)

drewverlee committed:
Update docs to mention adding target to paths.

drewverlee committed:
Update README.

drewverlee committed:
Remove extra space.

lambdatronic commented:
- 189,195: Maybe combine these two half sentences into just one sentence on 189 before the code block.

drewverlee committed:
Improve README.

By using ~ to mark files and code and connecting the what more directly to the why.
